README.md

```diff
@@ -1,2 +1,66 @@
-# capstone-01
-Capstone project 
+
+# Project Name
+- EA-GYM Project
+
+> A website for a GYM having multiple pages.
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- Bootstrap
+
+## Live Demo (if available)
+
+[Live Demo Link](https://eaesmat.github.io/capstone-01/)
+
+
+## Getting Started
+
+You can get started by cloning the project or vist the demo link
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+Have Git and GitHub
+### Setup
+Copy HTTPS link of the repo and clone it in your local machine.
+### Install
+Have a code editor to make changes.
+### Usage
+This project for a gym that any one can use it.
+### Run tests
+Run the linter before pushing the project.
+### Deployment
+This is already deployet just checkout the demo link
+
+
+## Authors
+
+👤 **Esmatullah Ahmadzai**
+
+- GitHub: [@githubhandle](https://github.com/eaesmat)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/esmatullah-ahmadzai-56bb9423b/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://eaesmat.github.io/capstone-01/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
```

Comment:
- [ ] Now your README file looks professional 👍, However, there is missing the appropriate credit to the author in your `readme` file. Please add it to make sure you follow the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requirements.
**Check the project requirements lines below:**

![image](https://user-images.githubusercontent.com/80612925/176719001-f0bc6c69-bb77-41ac-b70e-e6e8a7eaa2cb.png)